bin/get_nam_data.pl

```diff
@@ -411,7 +412,8 @@
       $jshash_ref->{"get"} = basename($0);
       $jshash_ref->{"filesDownloaded"} = \@files_downloaded;
       $jshash_ref->{"filesFromCache"} = \@files_found;
-      ASGSUtil::writeJSON($jshash_ref);
+      #ASGSUtil::writeJSON($jshash_ref);
```

Comment:
usually there's no reason to hide code behind a comment, but no change requested

Reply:
Sorry, leftover cruft ...